collect.py

```diff
@@ -10,13 +10,19 @@
 from PIL import Image
 from lxml import etree
 from selenium import webdriver
+from selenium.common.exceptions import ElementNotInteractableException
+from selenium.common.exceptions import InvalidSelectorException
 from selenium.common.exceptions import NoAlertPresentException
 from selenium.common.exceptions import NoSuchElementException
 from selenium.common.exceptions import NoSuchWindowException
+from selenium.common.exceptions import StaleElementReferenceException
 from selenium.common.exceptions import TimeoutException
+from selenium.common.exceptions import WebDriverException
 
 from autowebcompat import utils
 
+already_clicked_elems = set()
```

Comment:
We have several crawler threads, so this can't be a global variable (otherwise the different threads will overwrite each other's variable).